static/docker-compose.yml

```diff
@@ -80,6 +80,50 @@ services:
     volumes:
     - letsencrypt:/etc/letsencrypt:rw
 
+  notification-engine:
+    image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    depends_on:
+      - plextracdb
+      - redis
+    restart: always
+    environment:
+      CB_API_PASS: "${CB_API_PASS:?err}"
+      CB_API_USER: "${CB_API_USER:?err}"
+      COUCHBASE_URL: "http://plextracdb"
+      LOG_LEVEL: "${LOG_LEVEL:-error}"
+      REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
+      REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
+    healthcheck:
+      test: ["CMD", "npm", "run", "healthcheck:notification-engine", "readiness", "10"]
+      interval: 5s
+      timeout: 5s
+      retries: 3
+      start_period: 5s
+    entrypoint: npm run
+    command: "start:notification-engine"
+
+  notification-sender:
+    image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    depends_on:
+      - plextracdb
+      - redis
+    restart: always
+    environment:
+      CB_API_PASS: "${CB_API_PASS:?err}"
+      CB_API_USER: "${CB_API_USER:?err}"
+      COUCHBASE_URL: "http://plextracdb"
+      CLIENT_DOMAIN_NAME: ${CLIENT_DOMAIN_NAME:?err}
+      LOG_LEVEL: ${LOG_LEVEL:-error}
+      REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
+      REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
+      NOTIFICATION_DRY_RUN: ${NOTIFICATION_DRY_RUN}
+      serviceConfig: ${serviceConfig}
```

Comment:
Are these already available? I _think_ notification_dry_run may need added but I'm not familiar with serviceConfig.

Reply:
The `NOTIFICATION_DRY_RUN` env var does not need to be present - an 'undefined' value is treated as a `false` in code.